src/components/views/emojipicker/ReactionPicker.tsx

```diff
@@ -134,7 +134,7 @@ class ReactionPicker extends React.Component<IProps, IState> {
             isEmojiDisabled={this.isEmojiDisabled}
             selectedEmojis={this.state.selectedEmojis}
             showQuickReactions={true}
-            data-testid='mx_ReactionPicker'
+
```

Comment:
Why is there a blank line here?

Reply:
Because when I am struggling to get something simple working I start doing careless mistakes :p